docs/pages/versions/unversioned/sdk/screen-capture.mdx

```diff
@@ -128,6 +128,46 @@ export default function useScreenCaptureCallback() {
 
 </SnackInline>
 
+## Screen Recording Events
```

Comment:
```suggestion
### Example: Screen recording events
```

This should be part of the usage section, so let's convert it to an H3 where other examples are also listed:

![CleanShot 2024-12-10 at 23 09 36](https://github.com/user-attachments/assets/a495c6f5-c3b8-46b9-9cf7-7afa52b645ab)

Also, we use sentence casing instead of title for section titles.